config/default.json

```
@@ -11,6 +11,7 @@
     "files-scs:config/identity/handler/default.json",
     "files-scs:config/identity/ownership/token.json",
     "files-scs:config/identity/pod/static.json",
+    "files-scs:config/identity/registration/enabled.json",
```

Perhaps 
```suggestion
    "files-scs:config/identity/registration/default.json",
```
for consistency?

---

We have plenty of features where the default is not `default.json`. I usually only use default if there is only 1 valid option (that can be appended to), or if I really can't think of a better name for the option.

---

Perfect.